src/general/StageHUDBase.cs

```diff
@@ -213,12 +220,15 @@ public abstract class StageHUDBase<TStage> : Control, IStageHUD
     protected Panel environmentPanel = null!;
     protected GridContainer? environmentPanelBarContainer;
     protected ActionButton engulfHotkey = null!;
+    protected ActionButton secreteSlimeHotkey = null!;
     protected ActionButton signallingAgentsHotkey = null!;
 
     // Store these statefully for after player death
     protected float maxHP = 1.0f;
     protected float maxATP = 1.0f;
 
+    protected List<Compound> allAgents = new();
```

Comment:
Should this be a `readonly` variable? This seems like something that should not be reassigned.